src/services/contracts/acbuMinting.service.ts

```
@@ -101,6 +108,17 @@ export class MintingService {
         acbuAmount: acbuAmount.toString(),
       };
     } catch (error) {
+      if ((params as any).txId) {
+        try {
+          await prisma.transaction.update({
+            where: { id: (params as any).txId },
+            data: { status: "FAILED" }
```

ESLint

[error] Insert `,`

(prettier/prettier)